pkgs/kubevious/cli/registry.yaml

```diff
@@ -0,0 +1,14 @@
+packages:
+  - type: github_release
+    repo_owner: kubevious
+    repo_name: cli
+    asset: kubevious-{{.OS}}-{{.Arch}}
+    format: raw
+    description: Kubevious CLI - Prevent Kubernetes disasters at the early stages
+    files:
+      - name: kubevious
+        src: "{{.Asset}}"
```

Comment:
If `format` is `raw`, `files.src` isn't needed.
`files.src` is used to detect a file in an archived file.
In case of `raw` format, aqua can detect a file without `files.src`.

```suggestion
```